tests/acceptance/async_adapter_tests.py

```diff
@@ -2,9 +2,18 @@
 import uuid
 
 from pika import spec
+from pika import compat
 
 from async_test_base import (AsyncTestCase, BoundQueueTestCase, AsyncAdapters)
 
+
+def _p3_as_bytes(value):
```

Comment:
True, though actually the method in itself is a bit pointless, since ``as_bytes`` does what it does and as such the whole _p3_as_bytes -> as_bytes. I'll replace.